### PR TITLE
[packaging] Fix building without vendored make install macro. JB#55344

### PR DESCRIPTION
**Pull request by Thaodan**

### CODE REVIEW

File: rpm/libmlocale-qt5.spec [hunk 1]
```diff
@@ -52,7 +52,9 @@ export QT_SELECT=5
 
 %install
 export QT_SELECT=5
-%make_install
+%qmake5_install
+
+find %{buildroot} -name \*.a -delete
```
Comment (pvuorela) [inline rpm/libmlocale-qt5.spec:57]: There's --disable-static above. There shouldn't be anything static?

File: rpm/libmlocale-qt5.spec [hunk 2]
```diff
@@ -52,7 +52,9 @@ export QT_SELECT=5
 
 %install
 export QT_SELECT=5
-%make_install
+%qmake5_install
```
Comment (pvuorela) [inline rpm/libmlocale-qt5.spec:55]: %make_build but %qmake5_install feels asymmetrical?

Comment (pvuorela) [reply rpm/libmlocale-qt5.spec:55]: A bit pondering if in principle such %qmake5_install (or build) should even exist. Qmake builds are done by executing qmake which generates makefiles and the actual build and installation are done with make. For comparison there is no %automake_build or %automake_install.

Comment (Thaodan) [reply rpm/libmlocale-qt5.spec:55]: Please check the command that is executed in %qmake_install and how qmake defines things such as DESTDIR.
Qmake doesn't use the standard variable DESTDIR in `make install` but their own `INSTALL_ROOT` .
The `%qmake5_install` macro is there to handle these or other differences when needed, only the former is done at this moment.

File: rpm/libmlocale-qt5.spec (hunk 1 above)
Comment (Thaodan) [reply rpm/libmlocale-qt5.spec:57]: Please check the executed configure file. The original packager of this package assumed the used configure script is an autotools generated script however it is not.
There is no such argument.

Comment (abranson) [reply rpm/libmlocale-qt5.spec:57]: If there's an argument in the above call that does not exist in the script being called then it should be removed.

Comment (pvuorela) [reply rpm/libmlocale-qt5.spec:57]: That exactly ^

File: rpm/libmlocale-qt5.spec (hunk 2 above)
Comment (pvuorela) [reply rpm/libmlocale-qt5.spec:55]: Right, the overridden make_install used to define also that qmake variable. Okay then.

File: rpm/libmlocale-qt5.spec (hunk 1 above)
Comment (Thaodan) [reply rpm/libmlocale-qt5.spec:57]: Disable static is removed.

**Review by pvuorela — APPROVED** [review]

**Pull request merged**